Library/brew.sh

```diff
@@ -228,7 +228,8 @@ then
     -d av="$HOMEBREW_VERSION" \
     -d t=screenview \
     -d cd="$HOMEBREW_COMMAND" \
-    &
+    > /dev/null 2>&1 \
```

Comment:
`&> /dev/null` – nice! I wasn't aware that there's a shortcut for `> /dev/null 2>&1`.

I was hoping that `--output /dev/null` and `--silent` have a similar effect, but at least on my machine it doesn't cause the `curl` process to detach from `stdout`, thus blocking the parent process in the described circumstances.